### ISSUE TITLE
Non alphanumerical Names are displayed with ❓

### ISSUE THREAD

**Issue by martialblog** [Icinga/icinga-notifications-web#261]
Maybe there's a nicer solution then displaying a ❓ when a character is not found.

![screenshot_2024-07-25-130333](https://github.com/user-attachments/assets/a27a025a-cc71-4215-8c39-ef94b35259ac)

Not saying every Unicode character should be available, but maybe something nicer than ❓

**Comment by martialblog** [comment]
Also:
![screenshot_2024-07-25-130049](https://github.com/user-attachments/assets/271fea14-7022-400e-998b-a3d65038a3da)

But that's a whooooole different discussion I think.

Poor Boris. No mails for him

**Comment by martialblog** [comment]
A simple solution could be to use the generic 👤 instead for everyone. Similar to the Event Rule overview

**Comment by nilmerg** [comment]
Not sure what is causing this.

**Comment by julianbrost** [comment]
> Not sure what is causing this.

Random guess: does the code get the first character by simply taking the first byte of the string? If so, you'd get only part of an UTF-8 character and some function then used on the string would replace the invalid UTF-8 with �.

**Comment by martialblog** [comment]
Again, why not use a generic symbol instead of characters that might not even make sense in non-latin alphabets?

**Comment by nilmerg** [comment]
In case it's not possible to show a single non-ascii char (i.e. cyrillic), the user icon maybe is a better fallback. But using it for everyone is not ideal, because then it has no real advantage than showing nothing, which is also an option.

**Comment by julianbrost** [comment]
Yes, my guess was indeed correct. I've created #325 to solve that technical problem. If that's the way this is supposed to be fixed is up to @nilmerg to decide.

> instead of characters that might not even make sense in non-latin alphabets

Do you actually have an example/reference for this? (I'm not disputing the claim, I'd mainly be interested in learning more.)

**Comment by martialblog** [comment]
To be honest, I'm not sure myself, but that's kinda the point. The word "might" is very load-bearing here. I think the Arabic alphabet could be a candidate, maybe Japanese? 

I'm just saying why invest work in something that - as Johannes pointed out - "has no real advantage than showing nothing"
